Amazon.IonDotnet/Internals/Binary/SymbolTableReader.cs

```diff
@@ -536,11 +536,64 @@ public bool IsInStruct
         public int GetBytes(Span<byte> buffer)
             => throw new InvalidOperationException($"only valid if the value is a Lob, not {StateType(_currentState)}");
 
-        public IEnumerable<SymbolToken> GetTypeAnnotations()
+        public string[] GetTypeAnnotations()
```

Comment:
I've added implementation because I saw that the equivalent Java class had implementation here. I've held off removing this now to double check with you.

Reply:
Since it's not used, let's remove it for now. If there comes a time when it's needed, we can use this implementation and ensure we have the corresponding test coverage at that time.

Reply:
Thanks. Should these methods instead throw `NotImplementedException` instead? Prior, the default behaviour for `IEnumerable<SymbolToken> GetTypeAnnotationSymbols()` was to `yield break`. I could return an empty array for `string[] GetTypeAnnotations()` and return `false` for `bool HasAnnotation` if we want to not throw an exception in this case.

Reply:
I think throwing `NotImplementedException` is the best option.

Reply:
Fixed.